adoc/SAPDI3-SUSE-VMWare-VSAN-Install.adoc

```diff
@@ -134,12 +140,14 @@ After the installation workflow is successfully finished, you need to carry out
 
 ** Create a certificate request using `openssl`, for example:
 +
+[source, bash]
```

Comment:
Is the use of $ okay here in this command?
See line 143 - 153

Reply:
I missed to remove the $ here. Fixed with a new commit :)